english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md

```diff
@@ -0,0 +1,8 @@
+You suddenly travel to Berlin 1989, lying on the road, at night (without wearing cloth!!).
```

Comment:
You can check follow this flow: english/light-fire/xyzzy/stand-petrified/meet-wizard.md(i continue from this .md) , i add a link in his .md file that reader can jump to my .md file(path is : english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md)